azerothcore/azerothcore-wotlk
# fix(DB/broadcast_text): fix Stormwind guard Shaman trainer text

## Changes Proposed:
-  Fix the text in the dialogue when asking a guard for a Shaman trainer in Stormwind

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4656
- Closes https://github.com/chromiecraft/chromiecraft/issues/105

## SOURCE:
- I installed a mangos-one server to see what was in there, this is what I got:
![image](https://user-images.githubusercontent.com/24357633/116518453-87cdaf80-a8d0-11eb-9ec2-3a62a030113c.png)
- This matches an awesome [Wowhead comment from back in patch 3.3.3](https://www.wowhead.com/npc=68/stormwind-city-guard#comments:id=1024165):
<details>
  <summary>Comment contents saved for posterity</summary>
Class Trainer

Which class trainer are you looking for?

Druid
- The Park is where you want to go if you are looking for druid trainers. Theridran and Maldryn can usually be found by the moonwell in the center of the district.
- Well, you can find Theridan or Maldryn by the moonwell in the Park. Just head to the western area of Stormwind, north of the Mage District, but west of the Cathedral Square.

Hunter
- Thorfin Stoneshield came to us all the way from Ironforge to bring us his expertise in hunting. If you don't mind breathing the smoke filled air of the Dwarven District then you'll find him at his house in the northeast area of the district.
- A Hunter is it? Well, then you will want to speak with Thorfin Stoneshield over in the Dwarven District. Oh, and you be sure to tell him that if I step in his pet's business one more time, he's going to be training other hunters from behind bars in the Stockade!

Mage
- The magi congregate in the Wizard's Sanctum in the Mage Quarter. Quite a tower they have there. Makes our barracks look like peasant hovels.
- Sounds like you want to talk to Maginor Dumas! You can usually find him in the Wizard's Sanctum in the Mage Quarter. Course, you never know when or where those magi will portal off to.

Paladin
- If you are looking to become a mighty paladin, there is no better teacher then Lord Grayson Shadowbreaker. He can be found in the Cathedral of Light in Cathedral Square.

Priest
- I would suggest looking in the Cathedral of Light in Cathedral Square.
- Laurena is the High Priestess of Stormwind. You can find her in the Catheral of Light in Cathedral Square.

Rogue
- Old Town is where you should go if you want to find rogues. It's in the northeast corner of Stormwind.
- Bah, rogues have done nothing but make our job here in Stormwind harder. I wish I could drag both Osborne and Mathias both out of Old Town and right to the Stockade.

Shaman
- I don't know what's possessed you to look for a shaman... but I have seen someone who matches the profile. There's a woman who calls herself a Farseer on the shore in the Valley of Heroes.

Warlock
- Warlocks, eh. Well, you didn't hear it from me, but I've heard rumors that those types gather at a bar called the Slaughtered Lamb in the Mage Quarter.
- What? I don't know of any warlock trainers in Stormwind. Although... There have been some sightings of demonic activity over by The Slaughtered Lamb in the Mage Quarter. I suppose you could check there.

Warrior
- Check around Old Town at the Barracks or the Pig and Whistle Tavern. If you go to the tavern, be sure to sample some of the fine brews. Puts the dwarven ales to shame, I tell ya.
- Warriors can usually be found either at the Pig and Whistle Tavern or the Barracks in Old Town. Tell ya though, the tavern is probably a better place to look.
- Looking for some training, eh? I could train you better then anyone in Stormwind but the guard prohibits it. I guess you will have to settle for one of those that has been hanging out around the Barracks or at the Pig and Whistle Tavern in Old Town.
</details>

## Tests Performed:
- Applied the SQL fix
- Tested in-game on Windows 10 client and server


## How to Test the Changes:
- You *might* need to clear your client's cache, if it doesn't work the first time, try that
- Go to a female Stormwind City Guard (`.go c 79669`), ask about `Class Trainer` > `Shaman`, verify that she says `I don't know what's possessed you to look for a shaman... but I have seen someone who matches the profile. There's a woman who calls herself a Farseer on the shore in the Valley of Heroes.`
- Go to a male Stormwind City Guard (`.go c 241811`), ask about `Class Trainer` > `Shaman`, verify that he says `I don't know what's possessed you to look for a shaman... but I have seen someone who matches the profile. There's a woman who calls herself a Farseer on the shore in the Valley of Heroes.`

## Known Issues and TODO List:
None


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
